library/core/src/intrinsics/mod.rs

```diff
@@ -2995,7 +2995,7 @@ pub unsafe fn float_to_int_unchecked<Float: Copy, Int: Copy>(_value: Float) -> I
 
 /// Float addition that allows optimizations based on algebraic rules.
 ///
-/// This intrinsic does not have a stable counterpart.
+/// Stabilized as [`f16::algebraic_add`], [`f32::algebraic_add`], [`f64::algebraic_add`] and [`f128::algebraic_add`].
```

Comment:
Nit: the other docs have a consistent style for the message, this should probably also start with "The stable counterparts of this intrinsic are ...".

(Or "the public counterparts" like `select_unpredictable` since this isn't technically stable, but I think calling it "stable" is fine so we don't need to update it in the future).

Reply:
I copied `float_to_int_unchecked` directly above it since
```rust
/// Stabilized as ...
```
is nice and concise compared to
```rust
/// The stable counterparts of this intrinsic are ...
```

Should we either leave these as is or update `float_to_unchecked` to the more verbose wording too?

Reply:
Ah so I guess there are a few others, no need to update anything that is already there.